index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css?family=Arvo" rel="stylesheet" type="text/css">
+  <link href="styles/style.css" type="text/css" rel="stylesheet"/>
+</head>
+<div id="page">
```

Comment:
You're Index has no body.  Your page should have a head and a body.  The div you have named #page should be a body element instead of a div.  Also try and put your navigation inside of a nav element.